libbeat/cmd/instance/beat.go

```diff
@@ -883,7 +883,7 @@ func (b *Beat) checkElasticsearchVersion() {
 		if err != nil {
 			return err
 		}
-		if esVersion.LessThan(beatVersion) {
+		if esVersion.LessThanMajorMinor(beatVersion) {
```

Comment:
Shouldn't our comparison be LessThanOrEqual?

Reply:
No, we return an error if the version of ES is less than the version of Beats. If this was `LessThanOrEqual`, we would return an error when ES and Beats is on the same version.

Reply:
I compared the wrong way around 🤦‍♂️